src/Core/Settings.cs

```diff
@@ -284,6 +284,17 @@ public class ThemesImpl : SettingsOptionsAttribute.AbstractImpl
         [SettingsOptions(Impl = typeof(ThemesImpl))]
         public string Theme = "modern_dark";
 
+        public class HiddenTabsImpl : SettingsOptionsAttribute.AbstractImpl
+        {
+            public override string[] GetOptions => [.. Program.Web.RegisteredTabs.Keys];
+
+            public override string[] Names => Program.Web.RegisteredTabs.Values.Select(v => v.Name).ToArray();
```

Comment:
I believe Visual Studio will be highlighting this with a notice to swap it to `[.. ]` syntax?

Reply:
I was just copying from `RegisteredThemes`;

https://github.com/mcmonkeyprojects/SwarmUI/blob/3fba08da2fa766c80690a8da10cedcddb4df617f/src/Core/Settings.cs#L276-L281

VSCode .net extn's did not, which is weird since they did for a lot of other stuff. Asked copilot and changed it tho.